genai-microservices-connector/api/v1alpha3/gmconnector_types.go

```diff
@@ -0,0 +1,244 @@
+/*
+Copyright (c) 2024 Intel Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
The license header is different from the previous. Please align with Suyue.

Reply:
Try to use the same with https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/microservice/xeon/chatqna.py.  @chensuyue please let me know what is the correct license header should be used. Thanks~